pyseir/inference/model_fitter.py

```diff
@@ -210,9 +204,23 @@ def set_inference_parameters(self):
         """
         self.fit_params = self.DEFAULT_FIT_PARAMS
         # Update any state specific params.
-        for k, v in self.PARAM_SETS.items():
-            if self.state_obj.abbr in k:
-                self.fit_params.update(v)
+        initial_params_df = pd.read_csv("./pyseir/inference/world_according_to_pyseir.csv")
```

Comment:
I know I initially named this, but can we make it more meaningful. It should also be in the pyseir_data directory. So something like "pyseir_fitter_initial_conditions_2020_06_10.csv"

Reply:
And call the df something semantic like 'overwrite_params_df'?

Reply:
done